defaults/main.yml

```diff
@@ -7,7 +7,7 @@ fits_install_symlink: /opt/fits
 fits_user: tomcat8
 fits_group: tomcat8
 fits_ws: yes
-fits_log_path: /var/log/tomcat8
+fits_log_path: "{% if ansible_os_family == 'RedHat' %}/opt/tomcat/logs{% else %}/var/log/tomcat8{% endif %}"
```

Comment:
Instead of adding if logic here you can simply add the appropriate variable to the OS-Specific config files. See  https://github.com/yorkulibraries/ansible-role-fits/tree/issue-460/vars